packages/mdc-textfield/README.md

```diff
@@ -279,6 +279,20 @@ text, validation, and dense UI.
 </div>
 ```
 
+### Sass Mixins
+
+To customize a Filled Text Field or a Text Field `textarea`'s border radius, you can use the following mixins.
+Alternatively, if you would like to change the border radius for every instance of MDC Text Field uniformly, regardless
+of the variant, you can override $mdc-text-field-border-radius in Sass.
+
+#### `mdc-text-field-box-corner-radius($radius)`
+
+This mixin is provided for customizing the border radius for Text Field Box.
+
+#### `mdc-text-field-textarea-corner-radius($radius)`
+
+This mixin is provided for customizing the border radius for a Text Field `textarea`.
```

Comment:
Shorten? "Customizes the border radius for a Text Field `textarea`."

Reply:
Eh